pkg/tsdb/influxdb/models/datasource_info.go

```diff
@@ -9,7 +9,7 @@ type DatasourceInfo struct {
 	Token      string
 	URL        string
 
-	Database      string `json:"database"`
```

Comment:
hmm.. so, there already was code to read `jsonData.database`, but the value was never used?

Reply:
In `influxdb.go` line 181 it is being used `params.Set("db", dsInfo.Database)`

Reply:
i see, sorry, i got confuse because the same `models.DatasourceInfo` structure is created two times in `newInstanceSetting`.